Update setup.md (#103)
softwaresaved/az-intermediate-software-skills-course: setup.md

Once you have created a personal GitHub account (if you don't already have one), installed and configured your Git installation, and configured your proxy settings as above, to save time at the workshop please do the following to verify that you have done this correctly:

1. Using your personal GitHub account, create a public repository on GitHub.

> 2. If you are on an AstraZeneca Windows computer, move into your user directory (otherwise you will not have permission to write files)

Reviewer: ```suggestion
2. If you are on an AstraZeneca Windows computer, from GitBash navigate to your user directory (otherwise you may not have the permission to write files):
```
